docs/content/en/docs/operator-manual/piped/adding-an-analysis-provider.md

```diff
@@ -7,9 +7,17 @@ description: >
 ---
 
 
-You define the information needed to connect from your Piped to the Analysis Provider.
+To enable [Automated deployment analysis](/docs/user-guide/automated-deployment-analysis/) feature, you have to set the information needed to connect from your Piped to the [Analysis Providers](/docs/concepts/#analysis-provider).
 
```

Comment:
nit: `, you have to set the needed information for Piped to connect to the Analysis Provider.`

Reply:
Looks better 👍